### DISCUSSION TITLE
CHANNELD_AWAITING_LOCKIN for several weeks

### DISCUSSION THREAD

**Discussion by marcagra** [raspiblitz/raspiblitz#4360] [Q&A]
Hello, I'm new in the community and recently started to run a LN node. One month ago I tried to open my first 2 channels but they have not been confirmed yet. I'm even not be able to locate the funding transaction in the mempool. Please see below. Is there someone so kind to help me understand what happened and how to solve it? Thank you very much 

 "peers": [
      {
         "id": "035a46443fbf051aeb423f265fb8aa8589cdd607e............",
         "connected": true,
         "num_channels": 1,
         "netaddr": [
            "XX.XXX.XXX.XX"
         ],
         "features": "800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000888a52a1",
         "channels": [
            {
               "state": "CHANNELD_AWAITING_LOCKIN",
               "scratch_txid": "2e814d461a2545f0333341f8524062c59bf645e9169a7f7e6d6295f4b8682922",
               "last_tx_fee_msat": 183000,
               "feerate": {
                  "perkw": 253,
                  "perkb": 1012
               },
               "owner": "channeld",
               "direction": 0,
               "channel_id": "a5c08701de5861edbaa9db40d9dc1f41a80ab67e3a.......",
               "funding_txid": "39e5dc08bd5afcd4af333a3a7eb60aa8411fdcd940dba9baed6158de0187c0a5",
               "funding_outnum": 0,
               "close_to_addr": "bc1pfw33z2v35v5zyq9cqafzke3s6hp7ur0jqmj5.......",
               "close_to": "51204ba3112991a3282200b807522b6630d5c3ee0df206e5........",
               "private": false,
               "opener": "local",
               "alias": {
                  "local": "867322x1524342x63435"
               },
               "features": [
                  "option_static_remotekey"
               ],
               "funding": {
                  "local_funds_msat": 400000000,
                  "remote_funds_msat": 0,
                  "pushed_msat": 0
               },
               "to_us_msat": 400000000,
               "min_to_us_msat": 400000000,
               "max_to_us_msat": 400000000,
               "total_msat": 400000000,
               "fee_base_msat": 1000,
               "fee_proportional_millionths": 10,
               "dust_limit_msat": 546000,
               "max_total_htlc_in_msat": 18446744073709551615,
               "their_reserve_msat": 4000000,
               "our_reserve_msat": 4000000,
               "spendable_msat": 395460000,
               "receivable_msat": 0,
               "minimum_htlc_in_msat": 0,
               "minimum_htlc_out_msat": 1,
               "maximum_htlc_out_msat": 396000000,
               "their_to_self_delay": 144,
               "our_to_self_delay": 144,
               "max_accepted_htlcs": 30,
               "state_changes": [],
               "status": [
                  "CHANNELD_AWAITING_LOCKIN:Sent reestablish, waiting for theirs"
               ],
               "in_payments_offered": 0,
               "in_offered_msat": 0,
               "in_payments_fulfilled": 0,
               "in_fulfilled_msat": 0,
               "out_payments_offered": 0,
               "out_offered_msat": 0,
               "out_payments_fulfilled": 0,
               "out_fulfilled_msat": 0,
               "htlcs": []
            }
         ]
      },

**Comment by rootzoll** [comment]
If your opening transaction is no longer in the mempools you basically have two options: Try to rebroadcast it or make LND forget that you ever tried to open a channel and mark the UTXOs as usable again. 

For both ways there I dont know a out-of-the-box command and basically its a LND question not a RaspiBlitz question. But with high fee environment this might happen more often now - so if @guggero can give some directions here and what we can to on the RaspiBlitz in general to help people in those situations would be great.

**Comment by guggero** [comment]
This doesn't look like an `lnd` output, so I assume they're using CLN.

**Comment by rootzoll** [comment]
@guggero LOL right the the user wrote "LN node" node "LND node" - sliped my eye. Sorry for bothering.

@openoms do we have any any cln options here?

**Comment by openoms** [comment]
If the channel open does not go through in 2 weeks (2016 blocks) then the peers "forgets" and it can never be a lightning channel. The option there is to force close.

In this case it seems the funding tx was never confirmed, probably was pruned from the mempools due to low fee. In this case the onchain funds should be spent to make sure it can never confirm later.

Check:
```
lightning-cli listfunds
```
or RTL for the available UTXOs and if you are unsure which was used to open this channel spend all of them to a new address from CLN.

The pending channel might stay there until explicitly removed. Related discussion from a quick search: https://github.com/ElementsProject/lightning/issues/6504
